cpp/src/structure/induced_subgraph_impl.cuh

```diff
@@ -156,7 +155,10 @@ extract_induced_subgraphs(
       thrust::make_counting_iterator(size_t{0}),
       thrust::make_counting_iterator(num_aggregate_subgraph_vertices),
       subgraph_vertex_output_offsets.begin(),
-      [subgraph_offsets, subgraph_vertices, num_subgraphs, edge_partition] __device__(auto i) {
+      [subgraph_offsets,
+       subgraph_vertices,
+       num_subgraphs = subgraph_offsets.size() - 1,
```

Comment:
It seems like `num_subgraphs` is not used?

Reply:
Removed